src/Request/ParamConverter/AdminParamConverter.php

```diff
@@ -21,6 +21,11 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
+/**
+ * @deprecated since sonata-project/admin-bundle 3.x.
```

Comment:
Should we trigger deprecation somewhere ?

Reply:
To be honest, I don't know if we can check if the controller is type-hinted or not given the `ParamConverter`.